recipes/default.rb

```diff
@@ -1210,7 +1234,7 @@
 
 # We can't use the internal port yet as the certificate has not been generated yet
 hopsworks_certs "generate-int-certs" do
-  subject     "/CN=#{consul_helper.get_service_fqdn("hopsworks.glassfish")}/OU=0"
+  subject     "/CN=#{node['fqdn']}/L=glassfishinternal/OU=0"
```

Comment:
Is this supposed to be a different certificate than hopsworks.glassfish.service.consul?

Reply:
Yes, we have one certificate for the user `glassfish` and another used by the internal Glassfish HTTP interface.